less/index.less

```diff
@@ -48,12 +48,283 @@ table {
 	border-spacing: 0;
 }
 
-/* Set every element's box-sizing to border-box */
+ /*Set every element's box-sizing to border-box */
+
+
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	width: 100%;
+	font-size: 62.5%;
+
+}
+
+// Variables Here
```

Comment:
excellent variable use

Reply:
I'm not going to lie - I'm super digging being able to use variables. I'm super pattern oriented, and somewhat have trained my brain for it with past work - so this is awesome!